src/options.js

```diff
@@ -183,9 +183,13 @@ function populateObject (object, defaultValues) {
 }
 
 function getTests (options) {
-    var tests = readJSON(options.tests, defaults.tests);
-
-    if (!options.results && !options.resultIds) {
+    var tests = options.tests;
+	
+	if(! Array.isArray(tests)) {
```

Comment:
ahh I didn't catch this one. I thought I had updated everywhere I will fix this and start using DetectIndent